tracer/src/Datadog.Trace/ClrProfiler/AutomaticTracer.cs

```diff
@@ -17,6 +17,12 @@ internal class AutomaticTracer : CommonTracer, IAutomaticTracer, IDistributedTra
 
         private ICommonTracer _child;
 
+        IScope IDistributedTracer.GetActiveScope()
+        {
+            // The automatic tracer doesn't need to get the manual active trace
+            return null;
```

Comment:
Maybe this can be done in a separate PR, but there are several instances where the automatic side needs to know the Active scope, even if it's on the manual side:
- Getting the active trace_id/span_id for logs injection
- Places where `ActiveScope == null` checks are made to determine if the there is a root span or not

These places could currently use the `GetSpanContext` to get this, but it's rather costly to create a new object when all we need is to check that it's not null or to get a trace_id/span_id. Maybe we add another interface method to get the `IReadOnlyDictionary<string,string>` object as-is